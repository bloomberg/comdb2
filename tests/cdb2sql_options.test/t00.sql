select 'abc"efg';
select 'abc''efg';
select 'abc
efg';
select 'abc"efg';
select 'abc''efg';
select 'abc;efg';
