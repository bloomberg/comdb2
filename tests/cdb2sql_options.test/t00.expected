('abc"efg'='abc"efg')
('abc''efg'='abc'efg')
('abc
efg'='abc
efg')
('abc"efg'='abc"efg')
('abc''efg'='abc'efg')
('abc;efg'='abc;efg')
